convert2rhel/backup/certs.py

```diff
@@ -115,7 +116,7 @@ def enable(self):
             self.previously_installed = True
         else:
             try:
-                utils.mkdir_p(self._target_cert_dir)
+                files.mkdir_p(self._target_cert_dir)
```

Comment:
Can we maintain the same pattern and either import `mkdir_p` directly or use `files.mkdir_p`? I get that we are simply replacing with the new module, but maybe we could take the opportunity and decide which pattern we should use? 

This is minor, but thought to raise the question.

Reply:
Good point, I'll change it to be just `mkdir_p`